lightstep/converter.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Converter(object):
+    """Converter is a simple abstract interface for converting span data to wire compatible formats for the Satellites.
+    """
+
+    __metaclass__ = ABCMeta
```

Comment:
IIRC this won't work against Python 3.x - maybe use `six` to handle the metaclass case?

Reply:
hmm, the tests are running 2.7 and 3.4, is that a false positive?

Reply:
I also ran the trivial example with 3.4, worked without any issues.